frontend/src/main/java/com/google/tripmeout/frontend/storage/StorageBindingModule.java

```diff
@@ -0,0 +1,14 @@
+package com.google.tripmeout.frontend.storage;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.storage.InMemoryPlaceVisitStorage;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+
+public class StorageBindingModule extends AbstractModule {
```

Comment:
Can we call this the InMemory_ if it only provides in memory stuff? Ideally at some point we will an `InMemoryStorageModule` and a `DatastoreStorageModule` that can be wired depending on what we want to do (like local testing vs deploying).